testonly/hammer/hammer_test.go

```diff
@@ -124,8 +126,10 @@ func TestInProcessMapHammer(t *testing.T) {
 		LeafSize:      1000,
 		ExtraSize:     100,
 		// TODO(mhutchinson): Increase these when #1845 is understood & fixed.
-		MinLeaves:   100,
-		MaxLeaves:   150,
+		MinLeavesR:  100,
```

Comment:
async nit: The 2 configs seem the same. Maybe consolidate its creation into a shared helper?